### DISCUSSION TITLE
Inconsistency in stars URL/view

### DISCUSSION THREAD

**Discussion by MattIPv4** [community/community#8234] [Other Feature Feedback, Questions, & Ideas]
If I access stars from the nav profile dropdown, it takes me to the profile tab (even though its not a visible tab on profiles?): https://github.com/MattIPv4?tab=stars

If I then access a list from that view, it takes me to a stars sub-route, rather than the profile tab (this is a new page I believe): https://github.com/stars/MattIPv4/lists/a

If I remove the list slug from that route, which I'd expect to show me all my lists, it's a 404: https://github.com/stars/MattIPv4/lists

But if I remove the lists part of that path, and just access https://github.com/stars/MattIPv4, then this is a valid page with all my stars, but not my lists?

---

Imo, I like the new stars sub-route, and would love for everything to be handled under there rather than via the profile tab :)

**Comment by brianlovin** [comment]
We made a change to profiles yesterday so that `Stars` is now a top-level tab in the profile navigation — hopefully that helps here 😊 

I'll file an issue for the 404 on the lists route without a list slug.

**Reply by MattIPv4** [reply]
Hey Brian,

Awesome, thanks for the update! I do indeed see the tab on profiles now!

It still looks like https://github.com/MattIPv4?tab=stars and https://github.com/stars/MattIPv4 are two separate routes, which seems a bit odd to me -- given the profile tab, the former route seems to be correct, and the latter feels out of place?

**Reply by brianlovin** [reply]
Yes, agreed — I've opened an issue to redirect the `/stars/:login` route to the profile.
